### ISSUE TITLE
Demo template causes breaking error during project setup

### ISSUE THREAD

**Issue by devcrowley** [hyperbrew/bolt-cep#90]
If you choose the option to include the demo projects during the setup process, it fails.  If you choose the skeleton setup option, it continues as normal without errors.

The output is below:

```text
T   create-bolt-cep
│   by Hyper Brew | https://hyperbrew.co
|
o  Where do you want to create your project?
|  ./testingxxx
|
o  Which UI framework would you like to use?
|  React
|
o  Which React template would you like to start with?
|  Demo (setup for all apps (After Effects, Premiere, Photoshop, etc) with sample functions and buttons)
|
o  What do you want to use as your panel's display name?
|  Testingxxx
|
o  What do you want to use as your panel's id?
|  com.testingxxx.cep
|
o  Do you want to install dependencies? (recommended)
|  Yes
|
o  Installed  bolt-cep .
|
o  Installed dependencies via yarn.
|
o  Running initial build.
C:\Users\UserName\AppData\Local\Yarn\Data\global\node_modules\execa\lib\error.js:60
                error = new Error(message);

Error: Command failed with exit code 2: yarn build
error Command failed with exit code 2.
yarn run v1.22.19
$ rimraf dist/* && tsc -p "tsconfig-build.json" && vite build --watch false
src/js/main/main.tsx(48,5): error TS2555: Expected at least 2 arguments, but got 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
    at makeError (C:\Users\UserName\AppData\Local\Yarn\Data\global\node_modules\execa\lib\error.js:60:11)
    at handlePromise (C:\Users\UseName\AppData\Local\Yarn\Data\global\node_modules\execa\index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async runCommandInDirectory (C:\Users\UserName\AppData\Local\Yarn\Data\global\node_modules\create-bolt-cep\dist\lib\utils.js:28:9)
    at async buildBolt (C:\Users\UserName\AppData\Local\Yarn\Data\global\node_modules\create-bolt-cep\dist\lib\utils.js:10:5)
    at async main (C:\Users\UserName\AppData\Local\Yarn\Data\global\node_modules\create-bolt-cep\dist\index.js:47:9) {
  shortMessage: 'Command failed with exit code 2: yarn build',
  command: 'yarn build',
  escapedCommand: 'yarn build',
  exitCode: 2,
  signal: undefined,
  signalDescription: undefined,
  stdout: 'yarn run v1.22.19\n' +
    '$ rimraf dist/* && tsc -p "tsconfig-build.json" && vite build --watch false\n' +
    'src/js/main/main.tsx(48,5): error TS2555: Expected at least 2 arguments, but got 1.\r\n' +
    'info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.',
  stderr: 'error Command failed with exit code 2.',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
error Command failed.
Exit code: 1
Command: C:\Users\UserName\AppData\Local\Yarn\bin\create-bolt-cep
Arguments:
Directory: C:\development\illustrator_plugins
Output:
```

**Comment by justintaylor-dev** [comment]
Thank you for bringing this up

**Comment by justintaylor-dev** [comment]
Resolved in 639f8a11101f14da1eb4aaa0639c16455c6297e1
